gulp/tasks/scripts.js

```diff
@@ -27,32 +32,65 @@ gulp.task( 'scripts:modern', function() {
     } ) )
     .pipe( gulpUglify() )
     .pipe( gulpRename( 'modernizr.min.js' ) )
+    .on( 'error', handleErrors )
     .pipe( gulp.dest( paths.processed + '/js/' ) )
-    .pipe( webpackStream( webpackConfig ) )
+    .pipe( browserSync.reload( {
+      stream: true
+    } ) );
+}
+
+/**
+ * Bundle scripts in unprocessed/js/routes/
+ * and factor out common modules into common.js.
+ * @returns {PassThrough} A source stream.
+ */
+function scriptsModern() {
+  return gulp.src( paths.unprocessed + '/js/routes/common.js' )
+    .pipe( webpackStream( webpackConfig.modernConf ) )
     .on( 'error', handleErrors )
     .pipe( gulp.dest( paths.processed + '/js/routes/' ) )
     .pipe( browserSync.reload( {
       stream: true
     } ) );
-} );
+}
 
-gulp.task( 'scripts:ie', function() {
+/**
+ * Bundle IE9-specific script.
+ * @returns {PassThrough} A source stream.
+ */
+function scriptsIe() {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1796/commits/49335e3b7094033c1037c11fda4f773feb98ba0e